.yarnrc

```diff
@@ -0,0 +1,6 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
more on this here if you're interested https://yarnpkg.com/en/docs/cli/policies

Reply:
Thanks! That was the exact link I used to find out about --> yarn policies set-version

Reply:
oh i know, was leaving this for people who might wonder how the file originated